### PR TITLE
scap client: support building on suse

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/client/rubygem-foreman_scap_client/rubygem-foreman_scap_client.spec [hunk 1]
```diff
@@ -3,32 +3,49 @@
...
 %if 0%{?el6}
 Requires: ruby(abi)
 Requires: rubygem(json) >= 1.4
 Requires: rubygem(json) < 2.0
 %else
+%if 0%{?suse_version} == 0
 Requires: ruby(release)
 %endif
+%endif
 Requires: openscap-scanner
```
Comment (evgeni) [inline packages/client/rubygem-foreman_scap_client/rubygem-foreman_scap_client.spec:37]: that's `openscap-utils` on SUSE

File: packages/client/rubygem-foreman_scap_client/rubygem-foreman_scap_client.spec [hunk 2]
```diff
@@ -3,32 +3,49 @@
 
 %define rubyabi 1.8
```
Comment (ekohl) [inline packages/client/rubygem-foreman_scap_client/rubygem-foreman_scap_client.spec:4]: Is this still used?

**Review by ekohl — APPROVED** [review]
The generated files bit looks ok, so I trust you on this (after fixing repoclosure).

File: packages/client/rubygem-foreman_scap_client/rubygem-foreman_scap_client.spec (hunk 1 above)
Comment (ekohl) [reply packages/client/rubygem-foreman_scap_client/rubygem-foreman_scap_client.spec:37]:
```suggestion
%if 0%{?suse_version}
Requires: openscap-utils
%else
Requires: openscap-scanner
%endif
```

File: packages/client/rubygem-foreman_scap_client/rubygem-foreman_scap_client.spec (hunk 2 above)
Comment (evgeni) [reply packages/client/rubygem-foreman_scap_client/rubygem-foreman_scap_client.spec:4]: on el6, yes, but we don't build for that, so can be cleaned up I guess

Comment (ekohl) [reply packages/client/rubygem-foreman_scap_client/rubygem-foreman_scap_client.spec:4]: We still have other el6 macros, so we either clean them all up or leave them all in.

**Pull request merged**